src/pages/ADN/main.ts

```diff
@@ -0,0 +1,88 @@
+import { pageInterface } from '../pageInterface';
+
+export const ADN: pageInterface = {
+  name: 'ADN',
+  domain: 'https://animationdigitalnetwork.fr',
+  languages: ['French'],
+  type: 'anime',
+  isSyncPage(url) {
+    if ($('div[data-testid="video-player"]').length) return true;
```

Comment:
By using the Boolean operator here and every where else you dont need to use a if.
`return Boolean($('div[data-testid="video-player"]').length);`